src/glue/jobs/json_to_parquet.py

```diff
@@ -191,34 +191,34 @@ def drop_table_duplicates(
     table_data_type = table_name_components[1]
     spark_df = table.toDF()
     if "InsertedDate" in spark_df.columns:
-        sorted_spark_df = spark_df.sort(spark_df.InsertedDate.desc())
+        sorted_spark_df = spark_df.sort(
+            [spark_df.InsertedDate.desc(), spark_df.export_end_date.desc()]
```

Comment:
This was the fix for this jira. The rest of the changes in this file were auto formatting changes.